samples/nrf9160/nrf_cloud_mqtt_multi_service/boards/nrf9160dk_nrf9160_ns.overlay

```diff
@@ -0,0 +1,51 @@
+/ {
+	chosen {
+		nordic,pm-ext-flash = &mx25r64;
+	};
+};
+
+
+/	{
+	/delete-node/ leds;
+
+	leds0 {
+		compatible = "gpio-leds";
+		status = "okay";
+		label = "LED0";
+		led0: led_0 {
+			status = "okay";
+			gpios = <&gpio0 2 0>;
+			label = "Green LED 1";
+		};
+	};
+
+	leds1 {
+		compatible = "gpio-leds";
+		status = "okay";
+		label = "LED1";
+		led1: led_1 {
+			gpios = <&gpio0 3 0>;
+			label = "Green LED 2";
+		};
+	};
+
+	leds2 {
+		compatible = "gpio-leds";
+		status = "okay";
+		label = "LED2";
+		led2: led_2 {
+			gpios = <&gpio0 4 0>;
+			label = "Green LED 3";
+		};
+	};
+
+	leds3 {
+		compatible = "gpio-leds";
+		status = "okay";
+		label = "LED3";
+		led3: led_3 {
+			gpios = <&gpio0 5 0>;
+			label = "Green LED 4";
+		};
+	};
+};
```

Comment:
```suggestion
```

Is it necessary to delete and re-define the LED pins? I think leaving this blank will allow the defaults set in the global 9160dk device tree definitions to survive, or am I mistaken?

Reply:
I forget where I got the original copy of this file.  The LED stuff is irrelevant to my PR.  I'll remove it.

Reply:
Yep! That's right, ATv2 does this so that CAF can discover the LEDs and control them appropriately